### ISSUE TITLE
[Bug] Guild List will not refresh after import of csv

### ISSUE THREAD

**Issue by jmckinney88** [Alumian/CEPGP-Retail#277]
**The Error**
```
1x CEPGP\Utility.lua:750: attempt to call global 'CEPGP_UpdateGuildScrollBar' (a nil value)
CEPGP\Utility.lua:750: in function `CEPGP_populateFrame'
[string "*:OnClick"]:4: in function <[string "*:OnClick"]:1>

Locals:
items = nil
subframe = nil
tempItems = <table> {
}
total = nil
(*temporary) = nil
(*temporary) = "attempt to call global 'CEPGP_UpdateGuildScrollBar' (a nil value)"
L = <table> {
 High Priestess Jeklik = "High Priestess Jeklik"
 Gothik the Harvester = "Gothik the Harvester"
 Add Guild EP = "Add Guild EP"
 Buru the Gorger = "Buru the Gorger"
 Wushoolay = "Wushoolay"
 Lord Kri = "Lord Kri"
 Emperor Vek'nilash = "Emperor Vek'nilash"
 Ossirian the Unscarred = "Ossirian the Unscarred"
 Raid = "Raid"
 Golemagg the Incinerator = "Golemagg the Incinerator"
 Grand Widow Faerlina = "Grand Widow Faerlina"
 High Priestess Mar'li = "High Priestess Mar'li"
 Majordomo Executus = "Majordomo Executus"
 Gahz'ranka = "Gahz'ranka"
 Gehennas = "Gehennas"
 Taerar = "Taerar"
 Vaelastrasz the Corrupt = "Vaelastrasz the Corrupt"
 Thane Korth'azz = "Thane Korth'azz"
 High Priest Thekal = "High Priest Thekal"
 Noth the Plaguebringer = "Noth the Plaguebringer"
 Bloodlord Mandokir = "Bloodlord Mandokir"
 Moam = "Moam"
 Save Standings = "Save Standings"
 The Prophet Skeram = "The Prophet Skeram"
 Options = "Options"
 Flamegor = "Flamegor"
 C'Thun = "C'Thun"
 General Rajaxx = "General Rajaxx"
 The Edge of Madness = "The Edge of Madness"
 Princess Huhuran = "Princess Huhuran"
 Award to Hightest Roller = "Award to Hightest Roller"
 Azuregos = "Azuregos"
 Reset All EPGP Standings = "Reset All EPGP Standings"
 Kurinnaxx = "Kurinnaxx"
 Decay Guild EP = "Decay Guild EP"
 Ebonroc = "Ebonroc"
 View EPGP Traffic = "View EPGP Traffic"
 Import Standings = "Import Standings"
 Thaddius = "Thaddius"
 Loatheb = "Loatheb"
 The Silithid Royalty = "The Silithid Royalty"
 Lady Blaumeux = "Lady Blaumeux"
 Restore Standings = "Restore Standings"
 Raid Attendance = "Raid Attendance"
 Export Standings = "Export Standings"
 Teremus the Devourer = "Teremus the Devourer"
 Renataki = "Renataki"
 Highlord Mograine = "Highlord Mograine"
 Guild = "Guild"
 High Priest Venoxis = "High Priest Venoxis"
 The Twin Emperors = "The Twin Emperors"
 Heigan the Unclean = "Heigan the Unclean"
 %s has been defeated! Award %d EP to the raid? = "%s has been defeated! Award %d EP to the raid?"
 Onyxia = "Onyxia"
 %s has been defeated! %d EP has been awarded to the raid = "%s has been defeated! %d EP has been awarded to the raid"
 %s have been defeated! %d EP has been awarded to the raid = "%s have been defeated! %d EP has been awarded to the raid"
 Ysondre = "Ysondre"
 Lethon = "Lethon"
 Emeriss = "Emeriss"
 Baron Geddon = "Baron Geddon"
 Loot Distribution = "Loot Distribution"
 Sulfuron Harbinger = "Sulfuron Harbinger"
 Gri'lek = "Gri'lek"
 Add Raid EP = "Add Raid EP"
 Lord Kazzak = "Lord Kazzak"
 Decay Guild EPGP = "Decay Guild EPGP"
 Sapphiron = "Sapphiron"
 Gluth = "Gluth"
 Viscidus = "Viscidus"
 Patchwerk = "Patchwerk"
 Vem = "Vem"
 Ragnaros = "Ragnaros"
 Shazzrah = "Shazzrah"
 Ayamiss the Hunter = "Ayamiss the Hunter"
 Battleguard Sartura = "Battleguard Sartura"
 Ouro = "Ouro"
 Kel'Thuzad = "Kel'Thuzad"
 Nefarian = "Nefarian"
 Fankriss the Unyielding = "Fankriss the Unyielding"
 Lucifron = "Lucifron"
 Anub'Rekhan = "Anub'Rekhan"
 Hazza'rah = "Hazza'rah"
 Decay Guild GP = "Decay Guild GP"
 Firemaw = "Firemaw"
 Loot List = "Loot List"
 Maexxna = "Maexxna"
 Broodlord Lashlayer = "Broodlord Lashlayer"
 The Four Horsemen = "The Four Horsemen"
 Jin'do the Hexxer = "Jin'do the Hexxer"
 Grobbulus = "Grobbulus"
 Instructor Razuvious = "Instructor Razuvious"
 Razorgore the Untamed = "Razorgore the Untamed"
 Magmadar = "Magmadar"
 Sir Zeliek = "Sir Zeliek"
 Princess Yauj = "Princess Yauj"
 Emperor Vek'lor = "Emperor Vek'lor"
 Hakkar = "Hakkar"
 High Priestess Arlokk = "High Priestess Arlokk"
 Chromaggus = "Chromaggus"
 Garr = "Garr"
}
```

**Describe the bug**
The Guild List will not update beyond the base EP and GP values (0 and 26 for me).

**To Reproduce**
This bug started occurring after importing fresh data via CSV to convert from our previous loot system. After the import was complete, I expected the list to update. It failed to do so and clicking "guild" in the top left to refresh the list did nothing. After closing World of Warcraft Classic and relaunching, opening the window via `/cepgp show` and clicking "guild" yielded the above error in bugsack/swatter.

**Debug Information**
<details><summary>Debug Info</summary><br />
Version: 1.12.22 Release<br />
Locale: enUS<br />
Poll Rate: 0.0001<br />
GP Formula: (4.83x(2^<sup>((ilvl/26)+(rarity-4))</sup>)xSlot Modifier)x1<br />
Base GP: 26<br />
Traffic Entries: 241<br />
Base GP Decay Factor: false<br />
Standby EP: False<br />
Standby Offline: False<br />
Standby Percent: 100<br />
Standby Accept Whispers: False<br />
Standby EP by Rank: False<br />
Standby EP Manual Delegation: True<br />
Standby EP Whisper Keyphrase: standby<br />
Auto Pass on Ineligible Items: false<br />
Full Raid Loot Visibility: false<br />
GP on Tooltips: true<br />
Suppress Loot Announcements: false<br />
Roll for Passes: true<br />
Announce Rolls: true<br />
Resolve Duplicate Rolls: true<br />
Minimum EP: false, 0<br />
Reporting Channel: Raid<br />
Loot Response Channel: Raid<br />
Minimum Threshold for Loot: 3<br />
Allow Force Sync: false, Guild Master<br />
Hide Keyphrase Alternatives: False<br /><br /><details><summary>Auto EP</summary>
High Priestess Jeklik: True<br />
Ragnaros: True<br />
Buru the Gorger: True<br />
Ossirian the Unscarred: True<br />
Golemagg the Incinerator: True<br />
Grand Widow Faerlina: True<br />
High Priestess Mar'li: True<br />
Majordomo Executus: True<br />
High Priest Venoxis: True<br />
Gehennas: True<br />
Taerar: True<br />
Vaelastrasz the Corrupt: True<br />
High Priest Thekal: True<br />
Noth the Plaguebringer: True<br />
Lethon: True<br />
Moam: True<br />
The Prophet Skeram: True<br />
Flamegor: True<br />
C'Thun: True<br />
General Rajaxx: True<br />
The Edge of Madness: True<br />
Princess Huhuran: True<br />
Thaddius: True<br />
Loatheb: True<br />
The Silithid Royalty: True<br />
Ayamiss the Hunter: True<br />
Ebonroc: True<br />
Heigan the Unclean: True<br />
Azuregos: True<br />
Doom Lord Kazzak: True<br />
Onyxia: True<br />
Baron Geddon: True<br />
Viscidus: True<br />
Sulfuron Harbinger: True<br />
Gluth: True<br />
Jin'do the Hexxer: True<br />
Firemaw: True<br />
High Priestess Arlokk: True<br />
Ouro: True<br />
Shazzrah: True<br />
Gahz'ranka: True<br />
Battleguard Sartura: True<br />
Emeriss: True<br />
Kel'Thuzad: True<br />
Nefarian: True<br />
Fankriss the Unyielding: True<br />
Lucifron: True<br />
Anub'Rekhan: True<br />
Gothik the Harvester: True<br />
Kurinnaxx: True<br />
The Twin Emperors: True<br />
Hakkar: True<br />
Broodlord Lashlayer: True<br />
Maexxna: True<br />
The Four Horsemen: True<br />
Ysondre: True<br />
Grobbulus: True<br />
Instructor Razuvious: True<br />
Razorgore the Untamed: True<br />
Magmadar: True<br />
Patchwerk: True<br />
Bloodlord Mandokir: True<br />
Sapphiron: True<br />
Chromaggus: True<br />
Garr: True<br />
</details>
<details><summary>EP Values</summary>
High Priestess Jeklik: 2<br />
Ragnaros: 7<br />
Buru the Gorger: 3<br />
Ossirian the Unscarred: 4<br />
Golemagg the Incinerator: 5<br />
Grand Widow Faerlina: 12<br />
High Priestess Mar'li: 2<br />
Majordomo Executus: 5<br />
High Priest Venoxis: 2<br />
Gehennas: 5<br />
Taerar: 7<br />
Vaelastrasz the Corrupt: 7<br />
High Priest Thekal: 2<br />
Noth the Plaguebringer: 12<br />
Lethon: 7<br />
Moam: 3<br />
The Prophet Skeram: 10<br />
Flamegor: 7<br />
C'Thun: 12<br />
General Rajaxx: 3<br />
The Edge of Madness: 2<br />
Princess Huhuran: 10<br />
Thaddius: 15<br />
Loatheb: 15<br />
The Silithid Royalty: 10<br />
Ayamiss the Hunter: 3<br />
Ebonroc: 7<br />
Heigan the Unclean: 12<br />
Azuregos: 7<br />
Doom Lord Kazzak: 7<br />
Onyxia: 5<br />
Baron Geddon: 5<br />
Viscidus: 10<br />
Sulfuron Harbinger: 5<br />
Gluth: 12<br />
Jin'do the Hexxer: 2<br />
Firemaw: 7<br />
High Priestess Arlokk: 2<br />
Ouro: 10<br />
Shazzrah: 5<br />
Gahz'ranka: 2<br />
Battleguard Sartura: 10<br />
Emeriss: 7<br />
Kel'Thuzad: 15<br />
Nefarian: 10<br />
Fankriss the Unyielding: 10<br />
Lucifron: 5<br />
Anub'Rekhan: 12<br />
Gothik the Harvester: 12<br />
Kurinnaxx: 3<br />
The Twin Emperors: 10<br />
Hakkar: 3<br />
Broodlord Lashlayer: 7<br />
Maexxna: 15<br />
The Four Horsemen: 15<br />
Ysondre: 7<br />
Grobbulus: 12<br />
Instructor Razuvious: 12<br />
Razorgore the Untamed: 7<br />
Magmadar: 5<br />
Patchwerk: 12<br />
Bloodlord Mandokir: 2<br />
Sapphiron: 15<br />
Chromaggus: 7<br />
Garr: 5<br />
</details>
<details><summary>Loot Button Configuration</summary>
Loot Button 1<br />State: Enabled<br />Button Text: High Priority<br />Discount: 0%<br />Keyphrase Alternative: HP<br /><br />Loot Button 2<br />State: Enabled<br />Button Text: Low Priority<br />Discount: 100%<br />Keyphrase Alternative: LP<br /><br />Loot Button 3<br />State: Enabled<br />Button Text: Resist Set<br />Discount: 100%<br />Keyphrase Alternative: Resist<br /><br />Loot Button 4<br />State: Enabled<br />Button Text: Gbank D/E<br />Discount: 0%<br />Keyphrase Alternative: Bank<br /><br /></details><details><summary>Standby Guild Ranks</summary>
Guild Master: False<br />
Officer: False<br />
Officer Alt: False<br />
Raider: False<br />
Member: False<br />
Alt: False<br />
Llama: False<br />
Probie: False<br />
</details>
<details><summary>Slot Weights</summary>
2HWEAPON: 2<br />
NECK: 0.56<br />
HEAD: 1<br />
WEAPON: 1.5<br />
WRIST: 0.5<br />
WEAPONMAINHAND: 1.5<br />
SHIELD: 1.5<br />
HOLDABLE: 0.5<br />
RANGED: 2<br />
THROWN: 0.5<br />
FEET: 0.75<br />
RELIC: 0.5<br />
FINGER: 0.56<br />
WEAPONOFFHAND: 0.5<br />
TRINKET: 1.25<br />
WAND: 0.5<br />
CLOAK: 0.56<br />
CHEST: 1<br />
ROBE: 1<br />
HAND: 0.75<br />
WAIST: 0.75<br />
EXCEPTION: 1<br />
LEGS: 1<br />
SHOULDER: 0.75<br />
</details>
<details><summary>Override List</summary>
</details>
<details><summary>Addons List</summary>
ClickMacros<br />
AbarClassic<br />
AtlasLootClassic<br />
Auc-Advanced<br />
Auc-Filter-Basic<br />
Auc-Stat-Histogram<br />
Auc-Stat-iLevel<br />
Auc-Stat-Purchased<br />
Auc-Stat-Simple<br />
Auc-Stat-StdDev<br />
Auc-Util-FixAH<br />
BagBrother<br />
Bagnon<br />
Bartender4<br />
BeanCounter<br />
!BugGrabber<br />
BugSack<br />
CEPGP_ZA<br />
ChatLinkIcons<br />
CEPGP<br />
CEPGP-StandingsTracker<br />
ClassicCastbars<br />
ClassicCastbars_Options<br />
ClassicGuildBank<br />
Clique<br />
Decursive<br />
Details<br />
Details_Streamer<br />
Details_TinyThreat<br />
Enchantrix-Barker<br />
EPGP-Classic<br />
Gear to Gear Planner<br />
GTFO<br />
GuildInviteTool<br />
honorspy<br />
ItemRack<br />
LagBar<br />
LibStub<br />
LunaUnitFrames<br />
PallyPower<br />
RollTrackerClassic<br />
Stubby<br />
!Swatter<br />
TidyPlates_ThreatPlates<br />
ThreatClassic2<br />
TitanClassicAmmo<br />
TitanClassicBag<br />
TitanClassicClock<br />
TitanClassicGold<br />
TitanClassicLocation<br />
TitanClassicLootType<br />
TitanClassicPerformance<br />
TitanClassicRegen<br />
TitanClassicRepair<br />
TitanClassicVolume<br />
TitanClassicXP<br />
TitanClassic<br />
WeakAuras<br />
+Wowhead_Looter<br />
DejaClassicStats<br />
DBM-Core<br />
DBM-DefaultSkin<br />
DBM-StatusBarTimers<br />
</details>
<details><summary>Enabled Plugins</summary>
CEPGP-StandingsTracker<br />
CEPGP_ZA<br />
</details>
</details>

**Comment by jmckinney88** [comment]
Here is the import string:

```
Accerso,2042.18091,209.4821,
Airdreann,4938.469209,245.3128982,
AlsoTyrol,4948.296095,197.1444167,
Astriid,3112.559165,138.798649,
Boye,3134.388354,82.76477623,
Bruunor,4944.876831,275.7945774,
Chitilla,1537.709907,104.0963538,
Coloirraw,93.366,26,
Darkynhalvos,599.67,159.2,
Definity,4819.976061,95.49881963,
Demonicia,570.6975587,35.38091471,
Drchar,4700.740282,145.6569983,
Evalonne,4913.073244,194.2323088,
Finnie,0,26,
Freyah,4457.388602,179.9091279,
Gavise,4947.05731,172.1569968,
Geawnna,4292.682248,194.7601981,
Healusurass,4878.432364,114.0518623,
Hiroki,4323.740633,131.645314,
Hulkhogann,706.6992178,62.38188828,
Jaquiefrost,3221.729914,156.915646,
Jeepwar,4701.091503,214.3023245,
Jezzibelle,4933.041961,137.5213528,
Johar,4940.146602,85.63930674,
Kabam,4399.279466,117.7928677,
Manchu,4602.19926,95.15805013,
Markus,4883.564515,174.3752862,
Mazelle,404.8769981,76.18063895,
Merhild,3441.928726,154.9473321,
Nowadano,2327.972381,83.1651506,
Onirose,3350.65008,284.3350856,
Palgam,3715.83978,261.8843812,
Piratecat,4469.852612,171.8004576,
Purphoros,3759.200554,104.2303239,
Pyrulen,4952.714714,178.4217137,
Quaze,3469.247839,60.47885773,
Raist,726.74557,42.51163991,
Rentorar,4563.917713,194.4578779,
Resany,4290.572668,206.8344239,
Reshka,933.174,256.04,
Rhoril,1145.691,165.68,
Samzor,4942.239777,78.72831616,
Sorie,4949.36145,131.8521907,
Tanica,3647.662203,124.9494791,
Testecleaves,2028.489289,425.798909,
Trigrhappi,4298.828362,174.9479471,
Txtornader,4343.759798,295.8089642,
Vanagorn,4793.861051,155.4385369,
Velwyndris,601.776,112.58,
Zuraki,4195.864454,111.6220312,
```

**Comment by Alumian** [comment]
Thanks for the bug report and my apologies for not addressing this sooner. I've made some changes which should just floor out values with decimal places. I've just tested it and it seems to be working. It will be included in the next update.

**Comment by Alumian** [comment]
Resolved in 1.13.0.
